### DISCUSSION TITLE
Footer appears to the right due to an extra closing </div>

### DISCUSSION THREAD

**Discussion by hakonanes** [pradyunsg/furo#501] [Potential Issue]
I have a bibliography using [`sphinxcontrib-bibtex`](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/index.html) 2.4.2, and when built with Sphinx 5.1.1 using the Furo 2022.6.21 theme, the footer appears in the area where the right sidebar is (Contents):

![bilde](https://user-images.githubusercontent.com/12139781/183243490-f9d4339c-0bb3-4842-96b9-d1e146aedd4b.png)

This does not happen when using e.g. the Pydata theme:

![bilde](https://user-images.githubusercontent.com/12139781/183243516-508b6e5e-3c0e-4535-ab9a-78ce623007aa.png)

Looking at the `bibliography.html` file I find an extra closing `</div>`, that when commented, out solves the issue, and the footer appears at the bottom as expected. It shouldn't be there, and I don't know where it comes from, unfortunately. Note that the footer displays correctly on all other pages.

A permanent version of the docs with the footer misplaced, built from a PR by RTD: https://kikuchipy--555.org.readthedocs.build/en/555/bibliography.html.

**Comment by pradyunsg** [comment]
FWIW, I’m pretty sure that the issue arises from the extension, generating the wrong markup.

**Reply by hakonanes** [reply]
Thanks, I believe you are right.

I haven't managed to create a minimal working example showing this behaviour, so I don't know the cause of it... Feel free to close the discussion if it turns out no one else is experiencing this.

**Comment by umarcor** [comment]
FTR, I'm seeing the same issue. Comparison of a page with the footer at the bottom (left) and a page with the misplaced footer (right):

![image](https://user-images.githubusercontent.com/38422348/222833378-421452c5-481e-4ec9-b596-758a9afa1aeb.png)

**Comment by umarcor** [comment]
According to https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/309, this is a bug in docutils 0.18 and 0.19 which was fixed already (https://sourceforge.net/p/docutils/patches/195/) ,so it will be gone when a new release of docutils is published.
